sandboxed_containers/deploying-sandboxed-container-workloads.adoc

```diff
@@ -14,6 +14,29 @@ include::modules/sandboxed-containers-preparing-openshift-cluster.adoc[leveloffs
 .Additional resources
 * xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Installing a user-provisioned cluster on bare metal]
 
+[id="checking-nodes-sandboxed-containers"]
+== Checking if you can deploy {sandboxed-containers-first} in your cluster
+
+Before running {sandboxed-containers-first} containers, you need to check whether the nodes in your cluster are eligible to run Kata containers. Some cluster nodes might not comply with sandboxed containers' minimum requirements. The most common reason for node ineligibility is the lack of virtualization support on the node. If you attempt to run sandboxed workloads on ineligible nodes, you will experience errors. You can use the Node Feature Discovery Operator (NFD) and a `NodeFeatureDiscovery` resource to automatically check node eligibility. When installed, NFD prevents installation of the sandboxed runtime on ineligible nodes.
```

Comment:
I think it's better to remove this text "When installed, NFD prevents installation of the sandboxed runtime on ineligible nodes."  NFD doesn't prevent the installation. It's dependent on the `checkNodeEligibility` attribute.